its/ruling/src/test/resources/expected/python-S1172.json

```diff
@@ -1,46 +1,39 @@
 {
```

Comment:
Here the actual changes are mixed with the format change of the output json, sorry about that.